cftime/_cftime.pyx

```diff
@@ -1057,13 +1147,36 @@ Gregorial calendar.
         cdef datetime dt
         if isinstance(self, datetime) and isinstance(other, timedelta):
             dt = self
+            calendar = self.calendar
             delta = other
         elif isinstance(self, timedelta) and isinstance(other, datetime):
             dt = other
+            calendar = other.calendar
             delta = self
         else:
             return NotImplemented
-        return dt._add_timedelta(delta)
+        # return calendar-specific subclasses for backward compatbility,
+        # even though after 1.3.0 this is no longer necessary.
+        if calendar == '360_day':
+            #return dt.__class__(*add_timedelta_360_day(dt, delta),calendar=calendar)
```

Comment:
I left those there in case we decide to remove the calendar-specific subclasses.  They are really not needed, but I use them here just in case some is expected them to be returned by `__add__` and `__sub__` (since that was the previous behavior).